rfcs/SchemaCoordinates.md

```diff
@@ -0,0 +1,372 @@
+# RFC: Schema Coordinates
+
+**Proposed by:** [Mark Larah](https://twitter.com/mark_larah) - Yelp
+
+This RFC proposes formalizing "Schema Coordinates" - a human readable syntax to
+uniquely identify a type, field, or field argument defined in a GraphQL Schema.
```

Comment:
```suggestion
This RFC proposes formalizing "Schema Coordinates" - a human readable syntax to
uniquely identify a type, field, field argument, directive or directive argument defined in a GraphQL Schema.
```

Reply:
or enum, as joel suggested

Also "type" is unclear whether that means specifically the formal output `type` or is a generic term that captures input types as well. Perhaps there's a broader category we could capture here rather than trying to list the specifics exhaustively?

Reply:
"type" is a very broad term, the SDL `type Foo { ... }` is called an "object type" in most places. I think "type" is sufficient to cover enums (and scalars, input object types, unions, interfaces, tagged, etc), but yes "enum values" does need to be added :+1: I've edited the above.